src/types.ts

```diff
@@ -494,7 +494,9 @@ export interface RevolutCheckoutInstance {
    */
   createCardField: (options?: CardFieldOptions) => RevolutCheckoutCardField
   /** Accept payments via Revolut pay */
-  revolutPay?: (options: RevolutPayOptions) => RevolutCheckoutInstance
+  revolutPay: (options: RevolutPayOptions) => RevolutCheckoutInstance
+  /** Accept payments via open banking */
+  openBanking?: (options: CommonOptions) => RevolutCheckoutInstance
```

Comment:
Why this method is optional? :)

Reply:
It's hidden under the config within the widget right now e.g. this way it was only used in dev and not prod. It will be made fully available shortly and hence non optional i.e just like revolut pay was.